  <launch>
    <!-- Tracking Node-->
    <node pkg="homog_track" type="tracking_node" name="tracking_node" />
  </launch>
